R/Colombia.R

```diff
@@ -0,0 +1,76 @@
+#' Colombia Class for downloading, cleaning and processing notification data
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for Colombia
+#'
+#' @details Inherits from `DataClass`
+#' @source https://raw.githubusercontent.com/pcm-dpc/COVID-19/master/dati-regioni/dpc-covid19-ita-regioni.csv  # nolint
+#' @export
+#' @examples
+#' \dontrun{
+#' region <- Italy$new(verbose = TRUE, steps = TRUE, get = TRUE)
+#' region$return()
+#' }
+Colombia <- R6::R6Class("Colombia",
+  inherit = DataClass,
+  public = list(
+
+    # Core Attributes
+    #' @field country name of country to fetch data for
+    country = "Colombia",
+    #' @field supported_levels A list of supported levels.
+    supported_levels = list("1"),
+    #' @field supported_region_names A list of region names in order of level.
+    supported_region_names = list("1" = "departamento"),
+    #' @field supported_region_codes A list of region codes in order of level.
+    supported_region_codes = list("1" = "iso_3166_2"),
+    #' @field common_data_urls List of named links to raw data. The first, and
```

Comment:
For already implemented places don't need the bits of docs saying what needs to go here.

Reply:
The field names (#' @field ...)? Isn't this needed for the documentation?

Reply:
Not that bit. The bit saying this needs to be main is really documentation for those implementing datasets and isn't really meaningful for users.

Reply:
Got it, just remove the directional parts about what to put there. on it.